app/controllers/concerns/sign_in/authentication.rb

```diff
@@ -30,7 +30,7 @@ def load_user(skip_expiration_check: false)
       @current_user.present?
     rescue Errors::AccessTokenExpiredError => e
       render json: { errors: e }, status: :forbidden unless skip_expiration_check
-    rescue Errors::StandardError
+    rescue Errors::StandardError, MPI::Errors::AccountLockedError
```

Comment:
I think we want to remove it from here too, right?

Reply:
My understanding was that we only wanted to allow the unhandled 500 errors go through for `authenticate`, if we don't handle them for `load_user` then the "fail silently" part of the method no longer works. Is that what we want for `load_user` as well?

Reply:
The 'fail silently' is only important for users who aren't authenticated, cause then load user will receive one of the various 'access token not found, etc' errors, and it will return nil. If functions in the `User` model are being run it means that the user already authenticated, so at this point it doesn't matter much if it renders error or nil